packages/phone-number-privacy/combiner/test/end-to-end/get-contact-matches.test.ts

```diff
@@ -20,6 +21,15 @@ require('dotenv').config()
 jest.setTimeout(60000)
 
 describe('Running against a deployed service', () => {
+  xit('Service is deployed at correct version', async () => {
```

Comment:
is this test run under different conditions than above in ...combiner/test/end-to-end/get-blinded-sig.test.ts?

Reply:
Not really. We probably don't need these tests here at all to be honest since the status endpoint doesn't do anything. I'll remove them

Reply:
Ok! Fine with keeping one of them in though, could be useful to have it if/when the status endpoint is back in action? Up to you!